### PR TITLE
Stylings for the new header and footer

### PR DESCRIPTION
**Pull request by i0sea**
Updated stylings for header and footer. Some styles didn't port over and were made manually. This does not include the search bar nor mobile menu.

### CODE REVIEW

**Review by thecristen — CHANGES_REQUESTED** [review]
Thanks @i0sea. Can you remove the `/node_modules/.bin` folder that ended up in here?
Additionally, you've included multiple copies of the JS and CSS files here -- likely from running the mix task multiple times. Please delete all of the extras and keep the style linked from the `head.html` and the JavaScript linked from the `scripts.html`

File: header.html [hunk 1]
```diff
@@ -30,790 +44,1579 @@
...
-            <a rel="noopener" aria-controls="contactUs" class="desktop-nav-link js-header-link collapsed" data-parent="#desktop-menu" data-target="#contactUs" href="https://www.mbta.com/customer-support" role="tab">
-              <div class="nav-link-content js-header-link__content">
-                <span class="nav-link-name">
-                  Contact Us
+        </div>
+        <div class="m-menu__language">
+        </div>
+      </nav>
+  </div>
+  <nav class="m-menu--mobile" id="navmenu">
```
Comment (thecristen) [inline header.html:592]: Let's either try to make the mobile menu work (when clicking the "menu" button) or delete this menu entirely. Leaning toward deleting this nav

File: header.html [hunk 2]
```diff
@@ -30,790 +44,1579 @@
...
               </div>
             </div>
           </div>
-          <div class="nav-item">
-            <a rel="noopener" aria-controls="contactUs" class="desktop-nav-link js-header-link collapsed" data-parent="#desktop-menu" data-target="#contactUs" href="https://www.mbta.com/customer-support" role="tab">
-              <div class="nav-link-content js-header-link__content">
-                <span class="nav-link-name">
-                  Contact Us
+        </div>
+        <div class="m-menu__language">
```
Comment (thecristen) [inline header.html:588]: Ah interesting, I wouldn't expect this div to still be here

**Review by thecristen — CHANGES_REQUESTED** [review]
Thanks!!

There's a few minor style issues. Maybe it's worth seeing if we can import anything else into to the CSS file to fix it. See the spacing in the callout sections of the menu:

image

I don't know how we pulled it off but there's an extra menu button on smaller screens!

image

I'd like to keep the first Menu button (which happens to link to the /menu page) and remove the second one (which doesn't do anything)

**Comment by i0sea** [comment]
Curious why the spacing on the contact page and between the menu buttons was off even though those styles should've been imported from `variables.scss`. 🤔

**Comment by i0sea** [comment]
@thecristen  Delete all mobile menu buttons or just keep the one that links to the js menu, given that the js mobile menu isn't rendering?

**Comment by thecristen** [comment]
Keep the one linking to the JS menu!

**Comment by i0sea** [comment]
Removed extra mobile menu button, kept the functioning one. Imported core, app, and changed variables to vairiables.scss but the contact submenu remains the same 😖

**Pull request merged**